CMakeLists.txt

```diff
@@ -770,8 +765,9 @@ elseif(SWITCH_BUILD)
         POST_BUILD
         COMMAND ${DEVKITPRO}/tools/bin/nacptool --create "${SHORT_NAME}" "rsn8887" ${SWITCH_VERSION} ${SHORT_NAME}.nacp
         COMMAND ${DEVKITPRO}/tools/bin/elf2nro ${SHORT_NAME} ${SHORT_NAME}.nro --icon=${CMAKE_SOURCE_DIR}/res/switchdata/icon0.jpg --nacp=${SHORT_NAME}.nacp
-        COMMAND rm -rf release/${SHORT_NAME}
-        COMMAND mkdir -p release/${SHORT_NAME}
+        COMMAND rm -rf release
+        COMMAND mkdir release
+        COMMAND cd release && mkdir ${SHORT_NAME} && cd ..
```

Comment:
Why do we do this in two steps? `mkdir -p release/${SHORT_NAME}` will also create the parent `release` if it doesn't exist.

Reply:
Because doing it in one step doesn't work in Visual Studio as Windows doesn't support the `-p` parameter for `mkdir`.

This was the only way that I found which worked on both Visual Studio and Linux.